lib/senders/sms.js

```diff
@@ -162,3 +162,20 @@ function urlSafeBase64 (hex) {
     .replace(/\//g, '_')
     .replace(/=/g, '')
 }
+
+function startOfMinute (date) {
+  const year = date.getUTCFullYear()
+  const month = date.getUTCMonth() + 1
+  const day = date.getUTCDate()
+  const hour = date.getUTCHours()
+  const minute = date.getUTCMinutes()
+  return `${year}-${pad(month)}-${pad(day)}T${pad(hour)}:${pad(minute)}:00Z`
+}
+
+function pad (number) {
```

Comment:
Rather than implementing our own potentially-buggy version of this, have you considered using a battle-tested third-party library such as https://www.npmjs.com/package/left-pad?

Reply:
(in case there was any doubt, that's not a serious suggestion, just amusing myself...)

Reply:
actual lol